source/_theme/pydy/layout.html

```diff
@@ -1,10 +1,10 @@
 {% extends "basic/layout.html" %}
 
-{% set theme_css_files = ['_static/main.css', '_static/custom.css',
+{% set theme_css_files = ['/_static/main.css', '/_static/custom.css',
```

Comment:
Yes, only logo needs to be changed. I didn't knew that sphinx automatically handles headers. This is incorrect.